### PR TITLE
Create Flatpak manifest for com.github.dahenson.agenda

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

File: list-remove.patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
+--- ./src/AgendaWindow.vala
```
Comment (TingPing) [inline list-remove.patch]: Doesn't look like this patch is upstreamed?

File: com.github.dahenson.agenda.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	"app-id": "com.github.dahenson.agenda",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
+	"sdk": "org.gnome.Sdk",
+	"command": "com.github.dahenson.agenda",
+	"finish-args": [
+		"--socket=wayland",
+		"--socket=x11",
+		"--env=DCONF_USER_CONFIG_DIR=.config/dconf",		
+		"--own-name=com.github.dahenson.agenda",
```
Comment (TingPing) [inline com.github.dahenson.agenda.json]: Your app-id name is ownable by default.

File: list-remove.patch (hunk 1 above)
Comment (Alexander-Wilms) [inline list-remove.patch]: You're right. I'll create a pull request.

File: com.github.dahenson.agenda.json (hunk 2 above)
Comment (Alexander-Wilms) [inline com.github.dahenson.agenda.json]: Fixed

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/5/builds/400

Also please in the future fork from `new-pr` branch to keep a clean history.

**Comment by Alexander-Wilms** [comment]
Will do. All builds succeeded.

**Comment by Barthalion** [comment]
While on it, can you also ask if they're interested in co-maintaining this repository, like we already do with Lollypop?

**Comment by Alexander-Wilms** [comment]
Yeah, I wanted to send the mail as soon as the repo is created.

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/com.github.dahenson.agenda

**Pull request closed without merging**